views/js/controller/Delivery/monitoring.js

```diff
@@ -434,13 +434,33 @@ define([
                         reasonRequired: true,
                         categoriesSelector: cascadingComboBox(categories[actionName] || {})
                     });
+                    var warningAction;
+                    var warningMessage;
+                    var warningReasons;
 
                     if (!config.allowedResources.length) {
-                        if (_selection.length > 1) {
-                            feedback().warning(__('No report available for these test sessions'));
+                        if (config.deniedResources) {
+                            if (config.deniedResources.length > 1) {
+                                warningAction = __('Cannot') + ' ' + actionName + ' ' + __('these test sessions.');
+                                warningReasons += _.map(config.deniedResources, function (deniedResource, index) {
+                                    var opener = (index > 0 ? __('test') : __('Test')) + ' ';
+                                    return opener + deniedResource.id + ' ' + deniedResource.reason;
```

Comment:
The translate helper accept parametric sentences, so you should rather use this construction instead: `_('Test %s', deniedResource.id)`.

And if you have to adapt the sentence to the context, then you will have to call the translate helper for each case. The reason why is we use a scanner to gather all translations from source code, so the labels should be only simple atomic strings.